onadata/apps/api/models/organization_profile.py

```diff
@@ -101,6 +103,11 @@ def __str__(self):
         return u'%s[%s]' % (self.name, self.user.username)
 
     def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        # set avatar url to cache
+        # key should be <username>-org-avatar format
+        if 'avatar-url' in self.metadata:
+            cache.set('{}{}'.format(
+                self.user, ORG_AVATAR_CACHE), self.metadata['avatar-url'])
```

Comment:
- could you consider using [f-strings](https://realpython.com/python-f-strings/)?
- could we explicitly identify a field from the user object to use, instead of relying on the models `__str__` method? Based on the comment here that would be `self.user.username`
- do we need to separate the two strings somehow? e.g. `'{}-{}'` so that we can achieve what the comment says i.e. "<username>-org-avatar"